### PR TITLE
Read updater server URL from config file

### PR DESCRIPTION
**Pull request by LukasReschke**
Requires https://github.com/owncloud/core/pull/24500

### CODE REVIEW

File: src/Utils/Fetcher.php [hunk 1]
```diff
@@ -155,7 +155,13 @@ protected function getFeedUrl(){
 		$version['edition'] = $this->configReader->getEdition();
 		$version['build'] = $this->locator->getBuild();
 
-		$url = self::DEFAULT_BASE_URL . '?version=' . implode('x', $version);
+		// Read updater server URL from config
+		$updaterServerUrl = isset($this->configReader->get(['system'])['updater.server.url']) ? $this->configReader->get(['system'])['updater.server.url'] : '';
+		if($updaterServerUrl === '') {
```
Comment (blizzz) [inline src/Utils/Fetcher.php]: why don't you just call the get() once and check with either ! or empty()?

**Comment by blizzz** [comment]
One comment, but looks good otherwise :+1:

**Comment by PVince81** [comment]
@VicDeo second review ?

File: src/Utils/Fetcher.php (hunk 1 above)
Comment (VicDeo) [inline src/Utils/Fetcher.php]: @LukasReschke yes, executing get() once according to suggestion above will be faster.

**Comment by VicDeo** [comment]
@PVince81 Looks good now :cake: 
:+1:

**Comment by PVince81** [comment]
@VicDeo can you backport this to 9.0 ? Thanks

CC @cmonteroluque @DeepDiver1975

**Comment by VicDeo** [comment]
Stable9: https://github.com/owncloud/updater/pull/349

**Pull request merged**